app/routes/datamart/land.py

```diff
@@ -0,0 +1,171 @@
+"""Run analysis on registered datasets."""
+
+import random
+import uuid
+from uuid import UUID
+
+from fastapi import APIRouter, Depends, Path, Query, Request
+from fastapi.logger import logger
+from fastapi.openapi.models import APIKey
+from fastapi.responses import ORJSONResponse
+
+from app.models.pydantic.datamart import TreeCoverLossByDriverIn
+
+from ...authentication.api_keys import get_api_key
+from ...models.pydantic.responses import Response
+
+router = APIRouter()
+
+
+@router.get(
+    "/tree-cover-loss-by-driver",
+    response_class=ORJSONResponse,
+    response_model=Response,
+    tags=["Land"],
+)
+async def tree_cover_loss_by_driver_search(
+    *,
+    geostore_id: UUID = Query(..., title="Geostore ID"),
+    canopy_cover: int = Query(..., alias="canopy_cover", title="Canopy Cover Percent"),
+    request: Request,
+    api_key: APIKey = Depends(get_api_key),
+):
+    """Search if a resource exists for a given geostore and canopy cover."""
+    # create mock_ids state if it doesn't exist
+    if not hasattr(request.app.state, "mock_ids"):
+        request.app.state.mock_ids = {}
+
+    try:
+        resource_id = request.app.state.mock_ids[f"{geostore_id}_{canopy_cover}"]["id"]
+        return ORJSONResponse(
+            status_code=200,
+            content={
+                "status": "success",
+                "data": {"link": f"/v0/land/tree-cover-loss-by-driver/{resource_id}"},
```

Comment:
Should we make this a proper fully-qualified link? With the host etc.?

Reply:
@dmannarino I was unsure about too, I've seen APIs do both. It looks like RW gives fully qualified links, so I'll go with that for consistency

Reply:
Yes, I think we should have fully qualified links, too.